src/guides/v2.3/comp-mgr/upgrade-best-practices.md

```diff
@@ -0,0 +1,94 @@
+
+To help you plan and budget for upgrades you should monitor our [Release schedule]({{site.baseurl}}/release), you should plan upgrade tasks within your team's backlog ahead of time. Aim to complete this work with GA.
+
+*  Use the pre-release version to learn about each new release. Pre-release is General Availability code that is available to {{site.data.var.ee}} merchants and all partners two weeks before General Availability. If you have multiple stores, use the pre-release on your base store and verify that your custom modules and themes are compatible with it.
+
+*  Review the [Upgrade plan checklist](https://support.magento.com/hc/en-us/articles/360057968951) for {{site.data.var.ee}} to help you plan for your upgrade.
+
+*  Plan for upgrades at the beginning of the year. You will need to book a budget and resources to complete each upgrade. Use the upgrade cost estimation described above as a guide. Remember, the upgrade effort might vary significantly from project to project. Use your experiences and knowledge to make a plan as accurate as possible.
+
+*  If your upgrades are taking more effort than what we describe here, we recommend you audit your project and make adjustments to your environment to reduce the long-term maintenance.
+
+### Performing upgrades
+
+Upgrades should be done on regular basis, and under a predefined budget. We recommend scheduling pre-approved upgrades at the beginning of the year to ensure upgrades are planned and completed on time.
+
+Assess the work to be done for upgrading:
+
+*  Review the [release notes]({{page.baseurl}}/release-notes/bk-release-notes.html) to understand the scope and impact of the new version.
+
+*  Use the [Upgrade Compatibility Tool]({{site.url}}/upgrade-compatibility-tool/introduction.html) to identify potential problems that must be fixed in your custom code before attempting to upgrade to a newer version.
+
+*  If you are using 3rd-party extensions, validate their compatibility with the target version you are planning to upgrade to.
+
+### Post-upgrade testing
+
+Testing is the phase of an upgrade that requires more time. As a result, this process should be as automated as possible. You can benefit from using the core testing tools. The [Application Testing Guide]({{page.baseurl}}/test/testing.html) provides details.
+
+Use staging to test and validate your upgrade before moving to production.
+
+Make use of a **maintenance page**. Preparing this page in advance allows you to communicate with your customers, notifying them that work happening in the background. This page should be visible for a few minutes, but in case of a problem, you might need to use it longer. Having the appropriate content and design for your maintenance page will give your users a good experience even when your store is not available.
```

Comment:
```suggestion
Make use of a **maintenance page**. Preparing this page in advance allows you to communicate with your customers, notifying them that work is happening in the background. This page should be visible for a few minutes, but in case of a problem, you might need to use it longer. Having the appropriate content and design for your maintenance page will give your users a good experience even when your store is not available.
```